content/recipes/nestia.md

```diff
@@ -0,0 +1,698 @@
+### Nestia
+Automatic `SDK` and `Swagger` generator for the `NestJS`.
+
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/samchon/nestia/blob/master/LICENSE)&nbsp;[![npm version](https://badge.fury.io/js/nestia.svg)](https://www.npmjs.com/package/nestia)&nbsp;[![Downloads](https://img.shields.io/npm/dm/nestia.svg)](https://www.npmjs.com/package/nestia) &nbsp;[![Build Status](https://github.com/samchon/nestia/workflows/build/badge.svg)](https://github.com/samchon/nestia/actions?query=workflow%3Abuild)&nbsp;[![Guide Documents](https://img.shields.io/badge/wiki-documentation-forestgreen)](https://github.com/samchon/nestia/wiki) 
+
+  - Github: https://github.com/samchon/nestia
+  - NPM: https://www.npmjs.com/packages/nestia
+  - Guide Documents: https://github.com/samchon/nestia/wiki
+
+```bash
+# INSTALL NESTIA
+npm install --save-dev nestia
+
+# BUILDING SDK LIBRARY
+npx nestia sdk "src/controller" --out "src/api"
+npx nestia sdk "src/**/*.controller.ts" --out "src/api"
+npx nestia sdk "src/controller" \ 
+    --exclude "src/controller/test" \
+    --out "src/api"
+
+# BUILDING SWAGGER.JSON IS ALSO POSSIBLE
+npx nestia swagger "src/controller" -- out "swagger.json"
+```
+
+Don't write any swagger comment and DTO decorator. Just run the `nestia` up.
+
+  - No swagger comment/decorator
+  - No DTO comment/decorator
+  - Only pure `interface`s and NestJS code are required
+    - Support generic typed `interface`s and `controller`s
+    - Support union/intersection types
+    - Support conditional types
+
+The `nestia` is an evolved automatic `SDK` and `Swagger` generator than ever, who does not require any type of swagger comment or DTO decorator function, by analyzing your `NestJS` developed backend server code in the compilation level (`nestia` utilizes the TypeScript Compiler API).
+
+Therefore, don't write any swagger comment and don't use any DTO related decorator function. Just use the pure interface type with the pure `NestJS` code. Reading below sections and looking at example codes of the `nestia`, feel how the `nestia` is much stronger than the legacy `@nestjs/swagger`.
```

Comment:
> don't write any swagger comment and don't use any DTO related decorator function

This is the 4th time you've mentioned that swagger and class-validator decorators aren't necessary. It's a little redundant at this point. You could probably just merge this with the above paragraph and not make the mention again

Reply:
Also `@nestjs/swagger` isn't "legacy", it's a different approach. The package is still in constant development.